README.md

```diff
@@ -1,10 +1,45 @@
 # RIOPA.jl
 
 Reproducible Input Ouput (I/O) Pattern Application (RIOPA).
-Proxy app for I/O generation in the U.S. Department of Energy Exascale Computing Project using Julia.
+Proxy app for I/O generation in the U.S. Department of Energy Exascale Computing
+Project using Julia.
 
 Requires: 
 - [Julia](https://julialang.org/downloads/) v1.6 or later
 
 
+## Getting started
+It's helpful to set the environment variable `JULIA_MPI_PATH` to the top-level
+MPI installation directory. In addition, if `mpiexec` is not the proper run
+command for your system, set the environment variable `JULIA_MPI_EXEC` to the
+desired run command. See the MPI package
+[configuration](https://juliaparallel.github.io/MPI.jl/stable/configuration/)
+page for more options if necessary.
 
+From top-level RIOPA directory run
```

Comment:
We should specify these are general instructions. We might have separate sections for each OLCF system, e.g. Summit and Frontier (when available).

Reply:
The only differences are in how you set the environment variables. And I added explicit mention of srun and jsrun. I don't think we need any further instructions for summit or crusher.

Reply:
Agree